update.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash -e
 
 cd "$(dirname "$0")"
-HH="[0-3][0-9]"
+HH="[0-1][0-8]" # Ultra-Rapid releases with file suffixes 00,06,12,18
```

Comment:
if it isn't for matching any valid hour I probably wouldn't call it `HH`, but maybe it is fine to match all hour possibilities?

Reply:
I changed it to use all hours and kept the HH name